test/evolutionaryAlgorithm/SimulationComponents/Recombinator/impl/test_Recombinator.py

```diff
@@ -0,0 +1,34 @@
+from src.evolutionaryAlgorithm.SimulationComponents.Recombinator.impl.signlePointRecombinator import \
+    singlePointRecombinator
+from src.evolutionaryAlgorithm.SimulationComponents.Recombinator.impl.twoPointRecombination import \
+    twoPointRecombinator
+from src.evolutionaryAlgorithm.SimulationComponents.Recombinator.impl.averageRecombinator import \
+    averageRecombinator
+from src.evolutionaryAlgorithm.SimulationComponents.Individual.impl.FloatingPointIndividual import \
+    FloatingPointIndividual
+
+
+def test_Recombinator():
+    marriages = [[[1, 2, 3, 4], [5, 6, 7, 8]]]
```

Comment:
To nie jest lista list individual interface tylko lista floatów!

Reply:
Zmienione już